### DISCUSSION TITLE
Welcome to Auto Clicker - AutoFill

### DISCUSSION THREAD

**Discussion by Dhruv-Tech-Apps** [Dhruv-Techapps/acf-docs#357] [General]
If you are new to our extension follow below steps to get familiar with extension
- Check out [video on Youtube](https://youtu.be/M3yNuBkJW3g)
- Checkout [documentation](https://getautoclicker.com/docs/) on different feature and how to make use of it. 
- Try out our extension on [test page](https://test.getautoclicker.com/)
- Run into issue chat on [Discord](https://discord.gg/hArVQns) 

Few steps to try:
- Right click and try our record feature

Example to try
- [URL](https://getautoclicker.com/docs/3.x/configuration/url/) - https://test.getautoclicker.com/ ( on which you need extension to run)
- [Action](https://getautoclicker.com/docs/3.x/action/overview/) 1 : 
  - [Element Finder](https://getautoclicker.com/docs/3.x/action/element-finder/) - `//input[@name="username"]` (target element on which you need to perform action )
  - [Value](https://getautoclicker.com/docs/3.x/action/value/) - `Your name`
- [Action](https://getautoclicker.com/docs/3.x/action/overview/) 2 :
  - [Element Finder](https://getautoclicker.com/docs/3.x/action/element-finder/) - `//button[@random]`
  - [Value](https://getautoclicker.com/docs/3.x/action/value/) - leave it blank to trigger click
  
<hr/>

> [Configurations](https://configs.getautoclicker.com/configs) page to find different configuration

**Comment by vishal180618** [comment]
great extension, I went through the documentation and did not find the example to do the following

I want to parse a text and convert it into an int, do some mathematical operation, and will use this to fill an input field, How can I do this?

this is the HTML element:

`<div id="card-live-price" class="valign-wrapper vspace-between"><div class="contentPrimary bodyLargeHeavy"><div class="left-align" style="margin-bottom: 2px;">Company Name</div><div class="valign-wrapper contentSecondary bodySmallHeavy">₹1,912.55 (+1.18%)<div class="contentSecondary" style="margin-left: 8px;">Depth</div></div></div><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="currentColor" height="20" width="20" class="contentPrimary cur-po"><path fill="none" d="M0 "></path><path d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"></path></svg></div>`

can I convert ₹1,912.55 into integer

**Comment by Dhruv-Tech-Apps** [comment]
Use copy command with regex to copy only integer and user paste to paste
back value
*-----------------------------------------------------------------------------------------------*
*Docs: *Getting Started
<https://getautoclicker.com/docs/3.x/getting-started/introduction/> | Batch
<https://getautoclicker.com/docs/3.x/batch/overview/>| Action
<https://getautoclicker.com/docs/3.x/action/overview/>| Addon
<https://getautoclicker.com/docs/3.x/addon/overview/>| Settings
<https://getautoclicker.com/docs/3.x/settings/overview/>| Element Finder
<https://getautoclicker.com/docs/3.x/action/element-finder/>

Chat on Discord <https://discord.gg/vmnNfWKqnR>

Thanks and Regards,
Dhruv Techapps
